sql/wsrep_mysqld.h

```diff
@@ -478,6 +478,26 @@ void wsrep_deinit_server();
  */
 enum wsrep::streaming_context::fragment_unit wsrep_fragment_unit(ulong unit);
 
+/*
+ * Set encryption key. Serialize it and send to provider.
+ */
+bool wsrep_set_encryption_key(const void* key, size_t size,
+                              unsigned int version);
+
+/*
+ * Serialize key_version and key into buffer
+ */
+void wsrep_key_serialize(std::vector<unsigned char>& input,
```

Comment:
I can't find the function definition anywhere, should the declaration be removed?

Reply:
Functions are moved to wsrep_encryption.cc/h